perun/logic/stats.py

```diff
@@ -255,12 +261,10 @@ def delete_stats_file_across_versions(stats_filename, keep_directory=False):
     matches = []
     # Traverse all the version directories and attempt to delete the file
     for version, _ in list_stat_versions():
-        try:
+        # If the file was not found in this version, simply continue
+        with SuppressExceptions(exceptions.StatsFileNotFoundException):
```

Comment:
Suggestion: Maybe rename to `SuppressedExceptions`? Sounds more readable in with context.